container/cgroupfs/cpu.go

```diff
@@ -0,0 +1,149 @@
+// +build linux
+
+/*
+http://www.apache.org/licenses/LICENSE-2.0.txt
+
+
+Copyright 2015 Intel Corporation
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cgroupfs
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/intelsdi-x/snap-plugin-collector-docker/container"
+)
+
+// Cpu implements StatGetter interface
+type Cpu struct{}
+
+// GetStats reads throttling metrics from Cpu Group from cpu.stat
+func (cpu *Cpu) GetStats(stats *container.Statistics, opts container.GetStatOpt) error {
+	path, err := opts.GetStringValue("cgroup_path")
```

Comment:
This will complicate interface. For other groups then cgroups getters other sets of parameters is required. With opts it easier to handle different cases